experimentation/src/main/java/com/automattic/android/experimentation/remote/AssignmentsDtoMapper.kt

```diff
@@ -16,7 +16,7 @@ internal object AssignmentsDtoMapper {
                     Variation.Treatment(value)
```

Comment:
Minor (🔍): After seeing `FluxC` being removed everywhere, I search for it one more time and we get a final reference to it on this comment above, consider updating the comment so that it won't mislead someone thinking that this has anything to do with `FluxC`.

Reply:
I've adjusted the comment in 513eb17 . I left the `FluxC` name though, as I think "previous implementation" without any context is enigmatic and more difficult to look for. I hope this looks better now!

Reply:
Makes sense, thanks for updating the comment @wzieba ! 🙇